lib/lib_rss.php

```diff
@@ -52,7 +52,14 @@ function idn_to_puny($url) {
 		$parts = parse_url($url);
 		if (!empty($parts['host'])) {
 			$idn = $parts['host'];
-			$puny = idn_to_ascii($idn, 0, INTL_IDNA_VARIANT_UTS46);
+			// https://wiki.php.net/rfc/deprecate-and-remove-intl_idna_variant_2003
+			if (defined('INTL_IDNA_VARIANT_UTS46')) {
+				$puny = idn_to_ascii($idn, 0, INTL_IDNA_VARIANT_UTS46);
```

Comment:
Why don't you pass `IDNA_DEFAULT` constant as second argument instead of `0`? Here we don't know to what constant corresponds `0` unless by guessing or logging the `IDNA_DEFAULT` constant.

Reply:
Done 👍